### PR TITLE
First wave of flake8 linter changes

### PR DESCRIPTION
**Pull request by nth10sd**
This has the following changes:

* Split imports to multiple lines.
* Whitespace changes in rev efd9f13
  * Check by running `git show efd9f13 --color-words="[^[:space:]]|([[:alnum:]]|UTF_8_GUARD)+"`
    * No red lines (indicating non-whitespace changes) should appear
* Add missing newline.

Drops the flake8 error count (on rev e125ed7) by ~~**63.41%**~~ **85.48%**, from **1,812** to ~~**663**~~ **237**.

Let's get this in before the upcoming holidays since they are fairly trivial and change lots of files.

### CODE REVIEW

File: server/server/settings.py [hunk 1]
```diff
@@ -216,14 +220,14 @@ def resolver_context_processor(request):
 # If you are running FuzzManager behind a TLS loadbalancer,
 # uncomment the next line to let Django know that it should
 # behave as if we were using HTTPs.
-#SECURE_PROXY_SSL_HEADER = ('HTTP_X_FORWARDED_PROTO', 'https')
+# SECURE_PROXY_SSL_HEADER = ('HTTP_X_FORWARDED_PROTO', 'https')
```
Comment (choller) [inline server/server/settings.py]: I don't like this style change. Commented code shouldn't be prefixed with whitespace. Same applies to commented code in other files (not just settings). Can we disable this in the linter?

**Comment by nth10sd** [comment]
I looked through the list of E265 errors (12 in all - with message `block comment should start with '# '`) and reverted all of them. I also rebased to tip.

**Comment by nth10sd** [comment]
Including rebased commit rev c15378c76ddadf7790f894e58875e0dd0217434f but ignoring error types (E265 and F403), the updated error count is now decreased from **1,811** to **671**, a reduction of **62.95%**.

**Comment by nth10sd** [comment]
Pushed some more easy-pickings to this PR, fixing E501 "line too long" errors and renaming some variable names.

Error count now down by a further **408** for a total reduction so far of **85.48%**, leaving us with about **237** more.

**Comment by jschwartzentruber** [comment]
I mean adding the pylint/flake8 configs to tox.ini if necessary, in case that wasn't clear.

**Comment by nth10sd** [comment]
I only added flake8 configs to tox.ini for now. Anything pylint should be in a separate PR.

**Pull request merged**